### DISCUSSION TITLE
Does mrsk support podman?

### DISCUSSION THREAD

**Discussion by chazwatkins** [basecamp/kamal#392] [Q&A] [answered]
I'm looking to move from docker to podman.  Does msrk support using podman instead of docker?

**Comment by igor-alexandrov** [comment, marked as answer]
No, it doesn't.

**Reply by chazwatkins** [reply]
Thanks, @igor-alexandrov.  Is support for Podman planned for the future?

**Reply by igor-alexandrov** [reply]
To be honest, I don't know.

Can you tell more, why you decided to use it compared to Docker?

**Reply by chazwatkins** [reply]
When Docker switch Docker Desktop to paid for businesses, I switched to Podman Desktop.  There's no hard requirement to use Podman for deployments, it's only used for development.

**Reply by igor-alexandrov** [reply]
Got it.

**Comment by dhh** [comment]
Happy to see someone explore an adapter setup, so we can work with both Docker and podman. But the bar for added complexity is pretty low. So bear that in mind. It's not something we're going to explore at 37s.

**Reply by chazwatkins** [reply]
That's completely reasonable.  I appreciate the response.
